plugins/inputs/win_services/win_services.go

```diff
@@ -0,0 +1,138 @@
+// +build windows
+
+package win_services
+
+import (
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"golang.org/x/sys/windows/svc/mgr"
+    "fmt"
+)
+
+var sampleConfig = `
+  ## Names of the services to monitor. Leave empty to monitor all the available services on the host
+  service_names = [
+    "LanmanServer",
+    "TermService",
+  ]
+`
+
+var description = "Input plugin to report Windows services info: service name, display name, state, startup mode"
```

Comment:
Don't mention the specific fields, this will be enough: "Input plugin to report Windows services info"